src/style.css

```diff
@@ -0,0 +1,86 @@
+@font-face {
+  font-family: AvenirBook;
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: AvenirHeavy;
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.input_big {
+  height: 70px;
+  width: 100%;
+
+  font-family: Avenir-Book, Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+  padding: 23px 0 23px 62px;
+  box-sizing: border-box;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+}
+
+input:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.container_big {
+  margin-top: 20px;
+  width: 100%;
+  position: relative;
+  border-radius: 4px;
+
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+input:focus {
+  font-family: AvenirHeavy, sans-serif;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-sizing: border-box;
+  border-radius: 3px;
+  outline: none;
+  border: 1px solid #e1e7ed;
+}
+
+.icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+  color: #3d4e61;
+}
+
+.input_small {
+  height: 42px;
+  width: 100%;
+  font-family: Avenir-Book, Arial, Helvetica, sans-serif;
+  font-size: 14px;
+  color: #3d4e61;
+  padding-left: 33px;
+  line-height: 19px;
+  box-sizing: border-box;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+}
```

Comment:
check out point 2 in the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)